people/KenFitzsimmons.md

```diff
@@ -0,0 +1,4 @@
+## [Ken Fitzsimmons](KenFitzsimmons.md)
+### Projects
+#### [BitConnect](../projects/bitconnect.md) - platform for investment in BCC with interest
+##### Director
```

Comment:
Why all headers?  Check how it shows on https://github.com/VladimirLevv/base/blob/patch-5/people/KenFitzsimmons.md :(